source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -34,30 +34,25 @@ struct RollPitchYaw
     Eigen::Array3d rollPitchYaw;
 };
 
-struct Representations
-{
-    Eigen::AngleAxisd axisAngle;
-    Eigen::Vector3d rotationVector;
-    Eigen::Quaterniond quaternion;
-    std::array<RollPitchYaw, nofRotationConventions> rotations;
-};
-
 Eigen::Affine3d getTransformationMatrixFromYAML(const std::string &path);
 void saveTransformationMatrixToYAML(const Eigen::Affine3d &, const std::string &path);
 cv::Mat eigenToCv(const Eigen::MatrixXd &);
 Eigen::MatrixXd cvToEigen(const cv::Mat &);
 Eigen::Array3d rotationMatrixToRollPitchYaw(const Eigen::Matrix3d &rotationMatrix, const RotationConvention &rotation);
 Eigen::Matrix3d rollPitchYawToRotationMatrix(const Eigen::Array3d &rollPitchYaw, const RotationConvention &rotation);
+Eigen::Vector3d rotation_matrix_to_rotation_vector(const Eigen::Matrix3d &rotationMatrixs);
+std::array<RollPitchYaw, 4Ui64> rotation_matrix_to_list_of_roll_pitch_yaw(const Eigen::Matrix3d &rotationMatrix);
+Eigen::Matrix3d rotation_vector_to_rotation_matrix(Eigen::Vector3d rotation_vector);
+void list_of_roll_pitch_yaw_to_rotation_matrix(const std::array<RollPitchYaw, 4Ui64> &rpy_list);
```

Comment:
You're using `nofRotationConventions` in some places for the array length, I suggest doing that all over to avoid bugs if that number changes. Also note that `4Ui64` does not compile on GCC, it looks like some sort of Visual Studio extension maybe?